tests/interfaces/test_qnode_tf.py

```diff
@@ -22,6 +22,25 @@
 from pennylane import qnode, QNode
 from pennylane.tape import JacobianTape
 
+def test_import_error(mocker):

```

Comment:
How come this is moved outside of the `TestQnode` class? Perhaps it should be moved inside the class (otherwise de-indent the body, since it's currently indented too far).

Reply:
The original test was only on `diff_method="parameter-shift"`, and that passes.

But when I moved the test into `TestQnode`, it tests a variety of `diff_method` values, and the tests were failing. I don't know why it was failing when in the `TestQnode` class.   I'll try some further investigation.

Reply:
I moved it back into `TestQnode` but am skipping the `backprop` case.